src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -430,6 +430,28 @@ public void testImportUsersLargeList() {
     }
   }
 
+  @Test
+  public void testGetTenant() throws Exception {
+    TestResponseInterceptor interceptor = initializeAppForUserManagement(
+        TestUtils.loadResource("getTenant.json"));
+    Tenant tenant = FirebaseAuth.getInstance().getTenantManager().getTenantAsync("TENANT_1").get();
+    checkTenant(tenant, "TENANT_1");
+    checkRequestHeaders(interceptor);
+  }
+
+  @Test
+  public void testGetTenantWithNotFoundError() throws Exception {
+    initializeAppForUserManagement("{}");
```

Comment:
I believe this endpoint sends a 404 (or similar error response) when the tenant is not found. Lets update the test case accordingly. See https://github.com/firebase/firebase-admin-go/blob/e088f55036154f0cccf661992f854abd8fb1333f/auth/tenant_mgt_test.go#L1096-L1100

Reply:
I don't think I follow. The Node implementation suggests I check for a response missing a 'name' (which is the tenantId): https://github.com/firebase/firebase-admin-node/blob/2b952d43fd8784151ebdb60b65e88b9fd1888ccf/src/auth/auth-api-request.ts#L1463.

Reply:
That's just some additional validation they do on the "successful" responses. On missing tenants, the backend sends a 404 NOT_FOUND error.

Reply:
Done.